File: FootballApp/Models/Player.cs
using System;
using System.Collections.Generic;

namespace FootballApp.Models
{
    public class Player
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string Country { get; set; }
        public int PositionId { get; set; }
        public Position Position { get; set; }
        public int? TeamId { get; set; }
        public Team Team { get; set; }
        public string PhotoUrl { get; set; }
        public ICollection<PlayerLike> LikedUsers { get; set; }
    }
}

File: FootballApp-SPA/src/app/_models/Player.ts
import { Team } from './team';

export interface Player {
    id: number;
    name: string;
    surname: string;
    dateOfBirth: Date;
    country: string;
    position: string;
    number: number;
    teamId: number;
    team: Team;
    photoUrl: string;
}
